src/tests/Microsoft.PowerFx.Interpreter.Tests/FlagTests.cs

```diff
@@ -0,0 +1,43 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+using Microsoft.PowerFx.Core.Parser;
+using Microsoft.PowerFx.Preview;
+using Microsoft.PowerFx.Types;
+using Xunit;
+
+namespace Microsoft.PowerFx.Core.Tests
+{
+    public class FlagTests
+    {
+        [Theory]
+        [InlineData("Sum(A, A[@Value])")]
```

Comment:
>[InlineData("Sum(A, A[@Value])")] [](http://example.com/codeflow?start=8&length=33)

Can we add a comment explaining this ... 

"@ syntax is deprecated. Instead, users should write  ..... "